healpy/sphtfunc.py

```diff
@@ -943,6 +944,11 @@ def smoothing(
       If given, the directory where to find the weights data.
     verbose : bool, optional
       If True prints diagnostic information. Default: True
+    nest : bool, optional
+      If True, ordering scheme is NESTED. Default: False (RING)
+      The map ordering is modified by this function, the input map array should
```

Comment:
`reorder` makes copy of the array, so we are not modifying the input.
however I agree we need to mention that internally we are running `reorder`